deploy.cmd

```diff
@@ -1,6 +1,4 @@
-for /f %%D in ('wmic volume get DriveLetter^, Label ^| find "CIRCUITPY"') do set myDrive=%%D
-echo %myDrive%
-
+set myDrive=D:
```

Comment:
What happened here? Did deploy.cmd not find the CIRCUITPY drive on your machine? Did it show up with a different name?